src/cartridges/int_adyen_SFRA/cartridge/adyen/scripts/payments/adyenCheckout.js

```diff
@@ -244,8 +247,9 @@ function createPaymentRequest(args) {
       }
       paymentRequest.lineItems = AdyenGetOpenInvoiceData.getLineItems(args);
       if (
-        paymentRequest.paymentMethod.type.indexOf('ratepay') > -1 &&
-        session.privacy.ratePayFingerprint
+        paymentMethodType.indexOf('ratepay') > -1 &&
+        session.privacy.ratePayFingerprint &&
+        paymentMethodType.indexOf('riverty') === -1
```

Comment:
The condition "paymentMethodType.indexOf('riverty') === -1" is not required as we don't have paymentMethod.type value containing both ratePay and riverty